### DISCUSSION TITLE
[Proposal] Allow use `;` on noop constructor overloads

### DISCUSSION THREAD

**Discussion by nogic1008** [dotnet/csharplang#8913] [Language Ideas] [answered]
## Description

Allows constructor definitions that only call other constructors to be defined with a ; instead of an empty block.

```csharp
public class Sample(int id, string name)
{
    public Sample(): this(0, ""); // Use ";" instead of "{ }"
}
```

## Motivation

- `;` has fewer things to consider when formating compared to empty blocks. (new line, indent, whitespace...)
- Primary constructors make it common to write constructor overloads.

**Comment by 333fred** [comment, marked as answer]
Duplicate of https://github.com/dotnet/csharplang/discussions/4309.

**Reply by nogic1008** [reply]
It is hard to me to choose search keywords. Thanks a lot!
